tests/test_compile_shaders.py

```
@@ -548,4 +591,1705 @@
+    assert "ÜNICÖDÉ=1" in tasks[0][3]
+    assert "TËST=1" in tasks[0][3]
+
+
+@patch("hlslkit.compile_shaders.open")
+def test_parse_shader_configs_file_not_found(mock_open):
+    """Test parse_shader_configs with non-existent file."""
+    mock_open.side_effect = FileNotFoundError("File not found")
+    
+    try:
+        parse_shader_configs("nonexistent.yaml")
+        assert False, "Should have raised FileNotFoundError"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)